specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -7197,6 +7292,51 @@
       },
       "description": "A skill to split a string into chunks of text."
     },
+    "CustomEntityLookupSkill": {
+      "x-ms-discriminator-value": "#Microsoft.Skills.Text.CustomEntityLookupSkill",
+      "allOf": [
+        {
+          "$ref": "#/definitions/SearchIndexerSkill"
+        }
+      ],
+      "properties": {
+        "defaultLanguageCode": {
+          "$ref": "#/definitions/CustomEntityLookupSkillLanguage",
+          "description": "A value indicating which language code to use. Default is en."
+        },
+        "entitiesDefinitionUri": {
+          "type": "string",
+          "description": "Path to a JSON or CSV file containing all the target text to match against."
+        },
+        "inlineEntitiesDefinition": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CustomEntity"
+          },
+          "description": "The inline CustomEntity definition."
+        },
+        "globalDefaultCaseSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "A global flag for CaseSentitive. If CaseSensitive is not set in CustomEntity, this value will be the default value."
+        },
+        "globalDefaultAccentSensitive": {
+          "type": "boolean",
+          "x-nullable": true,
+          "description": "A global flag for AccentSensitive. If AccentSensitive is not set in CustomEntity, this value will be the default value."
+        },
+        "globalDefaultFuzzyEditDistance": {
+          "type": "integer",
+          "format": "int32",
+          "x-nullable": true,
+          "description": "A global flag for FuzzyEditDistance. If FuzzyEditDistance is not set in CustomEntity, this value will be the default value."
+        }
+      },
+      "externalDocs": {
+        "url": "https://docs.microsoft.com/en-us/azure/search/cognitive-search-skill-custom-entity-lookup"
+      },
+      "description": "A skill looks for text from a custom, user-defined list of words and phrases."
+    },
```

Comment:
No required values? I would assume that at least the inline or URI parameters need to be specified. You might add that to the description. Swagger can actually require one of the two, but I'm not sure our generators would handle that.